src/components/Settings/Speech/Speech.component.js

```diff
@@ -90,10 +98,14 @@ const Speech = ({
                 max={MAX_PITCH}
                 step={INCREMENT_PITCH}
                 onChange={handleChangePitch}
+                disabled={voice.voiceSource === 'cloud' ? true : false}
               />
             </div>
           </ListItem>
-          <ListItem aria-label={intl.formatMessage(messages.rate)}>
+          <ListItem
+            disabled={voice.voiceSource === 'cloud' ? true : false}
```

Comment:
if the user selects an online voice using offline mode `voice` would be undefined and this throws an error. We can prevent this checking `voice?.voiceSource === 'cloud`.
probably this isn't necessary if we prevent an undefined voice.